### ISSUE TITLE
fig_inches=(10,10) works, fig_inches=(10, 10) doesn't

### ISSUE THREAD

**Issue by basnijholt** [ioam/holoviews#77]
this works:
```python 
%%opts Layout [fig_inches=(10,10)]
hv.Image(np.random.randn(10,10)) + hv.Image(np.random.randn(10,10))
```

this doesn't:
```python
%%opts Layout [fig_inches=(10, 10)]
hv.Image(np.random.randn(10,10)) + hv.Image(np.random.randn(10,10))
```

Error:

```

Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/holoviews/ipython/display_hooks.py", line 203, in wrapped
    **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/holoviews/ipython/display_hooks.py", line 267, in layout_display
    layoutplot = LayoutPlot(layout, **opts(layout, get_plot_size(layout, size)))
  File "/usr/local/lib/python2.7/dist-packages/holoviews/plotting/plot.py", line 782, in __init__
    **dict(plotopts, **params))
  File "/usr/local/lib/python2.7/dist-packages/holoviews/plotting/plot.py", line 110, in __init__
    super(Plot, self).__init__(**params)
  File "/usr/local/lib/python2.7/dist-packages/param/parameterized.py", line 1021, in __init__
    self._setup_params(**params)
  File "/usr/local/lib/python2.7/dist-packages/param/parameterized.py", line 958, in override_initialization
    fn(parameterized_instance,*args,**kw)
  File "/usr/local/lib/python2.7/dist-packages/param/parameterized.py", line 1411, in _setup_params
    setattr(self,name,val)
  File "/usr/local/lib/python2.7/dist-packages/param/__init__.py", line 771, in __set__
    self._check(val)
  File "/usr/local/lib/python2.7/dist-packages/param/__init__.py", line 780, in _check
    super(NumericTuple, self)._check(val)
  File "/usr/local/lib/python2.7/dist-packages/param/__init__.py", line 763, in _check
    raise ValueError("Tuple '%s' only takes a tuple value."%self._attrib_name)
ValueError: Tuple 'fig_inches' only takes a tuple value.
```

**Comment by philippjfr** [comment]
That's an issue with the opts magic parser, it seems to get confused by spaces. Not entirely sure there's an easy fix.

**Comment by jlstevens** [comment]
I'm sure the parser could be improved but that would take a fair bit of thought, effort and time.

**Comment by jlstevens** [comment]
That said, I think one cheap trick would be to have a regular expression that removes the space internally as a preprocessing step before parsing.

 If we could strip the space from any tuple containing Python literals before trying to parse the result, that would probably go a fair way to fixing issues like this.  That said, a simpler approach like this wouldn't be sophisticated enough to handle spacing in nested tuples (for instance).

**Comment by jlstevens** [comment]
This particular issue should be fixed in f09e42e for the plot options (tested in 8eefaa4). Unfortunately, there are still some issues with spaces in the style options but I am fairly sure tuples and strings containing spaces are less common in the style options.

As this issue is resolved for now, I'l close it now. If there is another concrete example of a similar issue that persists (e.g a real example of a problem in the style options) please open a new issue.
